src/workers/python/papyros/papyros.py

```diff
@@ -101,6 +101,13 @@ def show():
 
         matplotlib.pyplot.show = show
 
+    async def install_imports(self, source_code, ignore_missing=True):
+        try:
+            await install_imports(source_code)
+        except ValueError:
```

Comment:
When is this raised?

Reply:
Mainly during autocompletion: when users are still typing and the autocomplete method is called, I try to install the imports   from what is currently present. If they are typing an import call, (e.g. import mat) when going for import math, this leads to trying to import the module 'mat', which leads to a ValueError.

Reply:
resolved